docs/index.md

```diff
@@ -67,7 +67,7 @@ spec:
             - name: DD_DOGSTATSD_URL
               value: "unix:///var/run/datadog/dsd.socket"
             - name: DD_TRACE_AGENT_URL
-              value: "/var/run/datadog/apm.socket"
+              value: "unix:///var/run/datadog/apm.socket"
```

Comment:
We looked into doing this back in October 2023:
- https://github.com/coopnorge/go-datadog-lib/pull/160

There is an issue related to this:
- https://github.com/coopnorge/go-datadog-lib/issues/58

Have we verified that this does not cause problems?